apps/wistia-videos/src/components/ConfigScreen.tsx

```diff
@@ -112,24 +102,26 @@ const Config = (props: ConfigProps) => {
   };
 
   const getProjects: () => void = async () => {
-    setLoadingStatus(true);
-    const response = await fetchProjects(parameters.apiBearerToken);
-    if (response.success) {
+    try {
+      setLoadingStatus(true);
+      const response = await fetchProjects(parameters.apiBearerToken || '');
       Notification.success('Your connection to the Wistia Data API is working.');
       // set the projects in the state (don't save all the projects in the config parameters to prevent
       // the config object to become very large and reach the size limit)
-      setFetchedProjects(response.projects);
+      setFetchedProjects(response);
       setShowButton(false);
       setRequiredMessage('');
-    } else {
-      Notification.error(`Connection to Wistia Data API failed: ${response.error}`);
-      console.log(`Connection to Wistia Data API failed: ${response.error}`);
-      setFetchedProjects([]);
-      if (response?.code && response?.error) {
-        setRequiredMessage(response?.error);
+      setLoadingStatus(false);
+    } catch (error) {
+      if (error instanceof WistiaError) {
+        Notification.error(`Connection to Wistia Data API failed: ${error.message}`);
+        console.log(`Connection to Wistia Data API failed: ${error.message}`);
```

Comment:
This was present in the code before so wasn't sure whether to remove it. Since we have the `Notification` above, I don't really think we need it so will remove.